charts/partials/templates/_topologyspreadconstraints.tpl

```diff
@@ -0,0 +1,23 @@
+{{ define "linkerd.topologySpreadConstraints" -}}
+{{- if .Values.enableTopologySpreadConstraints }}
+topologySpreadConstraints:
+- maxSkew: 1
+  topologyKey: failure-domain.beta.kubernetes.io/zone
```

Comment:
This label has been [declared deprecated](https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesioregion). Can you use `topology.kubernetes.io/zone` instead?